modules/rosa-adding-node-labels.adoc

```diff
@@ -109,9 +109,9 @@ $ rosa list machinepools --cluster=<cluster_name>
 .Example output
```

Comment:
Should we leverage rosa describe machinepool command instead of list machine pool? With all these fields added to machine pool, list command is stretched. With CLI 1.2.28, we release describe machine pool command with support for SG-ID.

Reply:
@bchandra-ocp I couldn't get describe machinepool to work locally - what version of the ROSA CLI is this available in?

Reply:
Just following up on the above comment with two things for completeness: (1) I've created an [issue](https://issues.redhat.com/browse/OSDOCS-8359) to review the machinepool listings and replace with describe where feasible, and (2) there are open issues to improve the listing under XCMSTRAT-39.